### PR TITLE
Prevent subscription service closing WebSocket connection

### PR DESCRIPTION
**Pull request by Sriep**
- [ ] I added unit tests for any code that added
- [ ] I updated the CHANGELOG.md 
- [x] All IP is original and not copied from another source
- [x] I assign all copyright to Loom Network for the code in the pull request

Fix for https://github.com/loomnetwork/loomchain/issues/1366

### CODE REVIEW

**Comment by eduardonunesp** [comment]
Thanks @Sriep  it fixes the issue

File: rpc/client.go [hunk 1]
```diff
@@ -62,9 +65,10 @@ func (c *Client) readPump(funcMap map[string]eth.RPCFunc, logger log.TMLogger) {
 	for {
 		_, message, err := c.conn.ReadMessage()
 		if err != nil {
-			logger.Error("websocket client read message error", "err", err)
-			websocket.IsUnexpectedCloseError(err, websocket.CloseGoingAway, websocket.CloseAbnormalClosure)
-			break
+			if websocket.IsUnexpectedCloseError(err, websocket.CloseGoingAway, websocket.CloseAbnormalClosure) {
```
Comment (Sriep) [inline rpc/client.go]: The existing code does not make sense, so I changed to what appears to be intended.

Comment (Sriep) [reply rpc/client.go]: Ok I moved this fix elsewhere

**Comment by eduardonunesp** [comment]
LGTM

**Pull request merged**